### PR TITLE
Sort CirlceCI config.yml keys to facilitate diff review after codegen

### PR DESCRIPTION
**Pull request by kostmo**
This will support another round of migration from hand-written configs to code generation.

### CODE REVIEW

File: .circleci/codegen_validation/overwrite_with_normalized.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+
+YAML_FILENAME=$1
+
+
+# Allows this script to be invoked from any directory:
+cd $(dirname "$0")
+
+pushd ..
+
```
Comment (malfet) [inline .circleci/codegen_validation/overwrite_with_normalized.sh:11]: Why two spaces between every block in `.sh` file?

File: .circleci/codegen_validation/normalize_yaml_fragment.py [hunk 2]
```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env python3
 
+import os
 import sys
+
+sys.path.append(os.path.join(sys.path[0], '..'))
+
```
Comment (malfet) [inline .circleci/codegen_validation/normalize_yaml_fragment.py:7]: Can you please add a comment, why this one is needed?

**Review by malfet — APPROVED** [review]

File: .circleci/codegen_validation/overwrite_with_normalized.sh (hunk 1 above)
Comment (kostmo) [reply .circleci/codegen_validation/overwrite_with_normalized.sh:9]: removed

File: .circleci/codegen_validation/normalize_yaml_fragment.py (hunk 2 above)
Comment (kostmo) [reply .circleci/codegen_validation/normalize_yaml_fragment.py]: added

**Review by seemethere — APPROVED** [review]

**Pull request closed without merging**